administrators.rst

```diff
@@ -759,6 +759,48 @@ Fields:
    the slash. Do so with caution and be prepared to put the leading-slash
    back if there is any impact on storing AIPs as a result.
 
+Debugging S3
+""""""""""""
+
+There are times when the S3 storage adapter needs to be debugged. For example,
```

Comment:
Maybe "may need to be debugged" would be more accurate? Should work out of the box for at least some folks.